agent/tests/api/source/test_elastic.py

```diff
@@ -25,7 +25,7 @@ class TestElastic:
                     'conf.queryInterval': 'seconds',
                 }
             }],
-            'er': b'[{"config":{"conf.httpUris":["http://es:9201"],"conf.index":"test1","conf.initialOffset":"now-3d/d","conf.isIncrementalMode":false,"conf.offsetField":"timestamp","conf.query":null,"conf.queryInterval":"seconds"},"name":"elastic","type":"elastic"}]\n'
+            'er': b'[{"config":{"conf.httpUris":["http://es:9201"],"conf.index":"test1","conf.initialOffset":"now-3d/d","conf.isIncrementalMode":false,"conf.offsetField":"timestamp","conf.queryInterval":"seconds"},"name":"elastic","type":"elastic"}]\n'
```

Comment:
could you please change the string here to array and compare it to result.json instead of result.data?